sw/device/lib/testing/lc_ctrl_testutils.h

```diff
@@ -0,0 +1,36 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_TESTING_LC_CTRL_TESTUTILS_H_
+#define OPENTITAN_SW_DEVICE_LIB_TESTING_LC_CTRL_TESTUTILS_H_
+
+#include <stdbool.h>
+
+#include "sw/device/lib/dif/dif_lc_ctrl.h"
+
+/**
+ * Lifecycle Controller handle.
+ */
+typedef struct lc_ctrl {
+  /**
+   * Lifecycle Controller peripheral interface.
+   */
+  dif_lc_ctrl_t dif;
+} lc_ctrl_t;
+
+/**
+ * Creates Lifecycle Controller handle and initializes the peripheral.
+ */
+lc_ctrl_t lc_ctrl_init(void);
+
+/**
+ * Checks whether Lifecycle Controller state has debug functions enabled.
+ *
+ * There could be implications for tests with debug functions enabled. For
+ * example, SRAM code execution is enabled when Lifecycle Controller is
+ * in one of these states and OTP_IFETCH is disabled.
+ */
+bool lc_ctrl_debug_func_enabled(const lc_ctrl_t *lc);
```

Comment:
Just rename to what was described above.

Reply:
Done.